cartridges/int_algolia_sfra/cartridge/templates/default/slots/product/frequently-bought-together.isml

```diff
@@ -0,0 +1,17 @@
+<isscript>
+    productIds = [];
+    var productIdsString = "";
+
+    if (session.privacy.algoliaAnchorProducts) {
+        productIdsString = session.privacy.algoliaAnchorProducts;
```

Comment:
I find it weird to rely on this variable which is only defined in the demo repo. Why can't we have the code that get the productIdsString here?

Reply:
This is a custom approach similar to Salesforce's Einstein recommendations anchor products. Some recommenders in Einstein require anchor products that must be explicitly defined and passed to content slots. Although there is a mechanism in SFCC background (`context-object` attribute) for Einstein anchor products, we were unable to use it and instead created a custom mechanism by creating a session variable. 

To better understand this, please refer to the [Salesforce documentation](https://help.salesforce.com/s/articleView?id=cc.b2c_optimize_code_for_multi_anchor_recommendations.htm&type=5) and my [draft PR](https://github.com/algolia/doc/pull/8879) for doc. If it's still unclear, please let me know, and I will explain it further.

Reply:
`pdict.contextAUID` is not available for all pages, it is only available for some specific pages

Reply:
Ok I see, too bad